linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ESFeatures.scala

```diff
@@ -71,6 +73,35 @@ final class ESFeatures private (
    */
   val allowBigIntsForLongs = _allowBigIntsForLongs
 
+  /** Avoid `class`'es when using `function`s and `prototype`s has the same
+   *  observable semantics.
+   *
+   *  Default: `true`
+   *
+   *  SpiderMonkey is known to exhibit terrible performance with JavaScript
+   *  `class`'es, with up to an order of magnitude of performance degradation.
```

Comment:
I assume we don't have an easy reference for this? Otherwise add it?

Reply:
No, I did not find any existing reference about that. But when I get more time to do it properly, I will submit an issue to SpiderMonkey.